test/lib/jdk/test/whitebox/WhiteBox.java

```diff
@@ -560,6 +560,9 @@ public void clearInlineCaches(boolean preserve_static_stubs) {
   // Force Full GC
   public native void fullGC();
 
+  // Force a controlled crash (debug builds only)
+  public native void controlledCrash(int how);
```

Comment:
Please put this somewhere other than in the middle of GC & reference processing related functions.